src/promptflow-core/promptflow/executor/flow_executor.py

```diff
@@ -1155,8 +1157,15 @@ def _stringify_generator_output(self, outputs: dict):
             if isinstance(v, GeneratorType):
                 outputs[k] = "".join(str(chuck) for chuck in v)
 
+            if isinstance(v, AsyncGeneratorType):
+                outputs[k] = asyncio.run(self._stringify_async_generator(v))
```

Comment:
Does this work for real openai calls?

Reply:
What do you refer by working for real openai calls? Are you suggesting the output should be the content of the ChatCompletionChunk text field?